src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -946,6 +946,178 @@ public void TestConnectionStringEnvVarHasHighestPrecedence()
             }
         }
 
+        /// <summary>
+        /// Validates the error message that is returned for REST requests with incorrect parameter type
```

Comment:
Can rephrase it to a short and concise summary:

Test to validate that when a parameter type is incorrectly configured in a REST request in production mode, a generic error message is returned in the exception, thus preventing leakage of information about underlying database to the user.